### PR TITLE
Account Screen update

### PR DESCRIPTION
**Pull request by cristhianescobar**
Screenshot 2024-01-21 at 1 29 06 AM
Screenshot 2024-01-21 at 1 29 27 AM

### CODE REVIEW

**Comment by martyu** [comment]
Should iOS look like this too?

File: android/core/resources/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,75 +1,81 @@
...
-    <string name="tribe">Tribe</string>
-    <string name="stars">Stars</string>
-    <string name="wallet">Wallet</string>
+    <string name="library_albums">YOUR ALBUMS</string>
+    <string name="library_artists">YOUR ARTISTS</string>
+    <string name="library_download">Download</string>
+    <string name="library_empty_content_subtitle">Time to rescue it with your epic music stash! 🎶</string>
+    <string name="library_empty_content_title">Your library is empty.</string>
+    <string name="library_search">Search</string>
+    <string name="logout_dialog_message">Are you sure you want to logout?</string>
```
Comment (martyu) [inline android/core/resources/src/main/res/values/strings.xml]:
```suggestion
    <string name="logout_dialog_message">Are you sure you want to log out?</string>
```

File: android/core/resources/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,75 +1,81 @@
...
+    <string name="logout_dialog_title">👋 %s not so fast!</string>
```
Comment (martyu) [inline android/core/resources/src/main/res/values/strings.xml]:
```suggestion
    <string name="logout_dialog_title">👋 %s, not so fast!</string>
```

Comment (martyu) [reply android/core/resources/src/main/res/values/strings.xml]: why are we waving goodbye as we're trying to stop them from leaving...? 😸

File: shared/build.gradle.kts [hunk 3]
```diff
@@ -33,7 +33,7 @@ kotlin {
 
     val xcf = XCFramework()
     listOf(
-//        iosX64(),
+        iosX64(),
```
Comment (martyu) [inline shared/build.gradle.kts:36]: Did you mean to uncomment this?

**Review by martyu — DISMISSED** [review]

**Comment by martyu** [comment]
two little grammar fixes then looks good

File: android/core/resources/src/main/res/values/strings.xml (hunk 2 above)
Comment (cristhianescobar) [reply android/core/resources/src/main/res/values/strings.xml]: I just put something together for the internal team to see, the copy might change.

File: shared/build.gradle.kts (hunk 3 above)
Comment (cristhianescobar) [reply shared/build.gradle.kts:36]: Yeah, was cleaning up the share gradle file and since we need this long term I put it back.

**Comment by cristhianescobar** [comment]
> Should iOS look like this too?

It would be nice, I basically just put something together for the team to see.

**Review by newmskywalker — APPROVED** [review]

**Pull request merged**